infinite_tracing/tasks/proto.rake

```diff
@@ -3,32 +3,12 @@
 # See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
+require_relative '/helpers/license'
```

Comment:
You can drop the leading slash here:

```diff
- require_relative '/helpers/license'
+ require_relative 'helpers/license'
```

Reply:
I need to remember this it's like the 5th time you've helped out on this type of edit. [Done](https://github.com/newrelic/newrelic-ruby-agent/pull/1421/commits/e3c43a60bacc3b12f6541cd95d0aab5b0d2ee276)!